synapse/models/inet.py

```diff
@@ -411,6 +410,45 @@ def _revModl201708141516(self):
         if 'syn:type:fields' not in node[1]:
             self.core.addRows([(node[0], 'syn:type:fields', 'url=inet:url,file=file:bytes', s_common.now())])
 
+    @modelrev('inet', 201708172326)
+    def _revModl201708141516(self):
+
+        props = [
+            ('user',
+             {'ptype': 'inet:netuser', 'doc': 'The netuser associated with the logon event.', 'ro': 1,
+              'form': 'inet:netlogon'}),
+            ('time',
+             {'ptype': 'time', 'doc': 'The time the netuser logged into the service', 'ro': 1,
+              'form': 'inet:netlogon'}),
+            ('status', {'ptype': 'bool', 'ro': 1,
+                        'doc': 'The status of the logon action, denoting if it was successful or not.',
+                        'form': 'inet:netlogon'}),
+            ('ipv4', {'ptype': 'inet:ipv4', 'doc': 'The source IPv4 address of the logon.', 'form': 'inet:netlogon'}),
+            ('ipv6', {'ptype': 'inet:ipv6', 'doc': 'The source IPv6 address of the logon.', 'form': 'inet:netlogon'}),
+            ('logout', {'ptype': 'time', 'doc': 'The time the netuser logged out of the service.',
```

Comment:
`logout` might be hard to use here. If I form a node when a user logs in, how do I know what node to update? What if the user logs in, then logs in again (another session, at the exact same time), then logs out of the first session?

I'm thinking that maybe we want a field that tracks whether it was a log in or out and to have a guid as the primary property. Use case in my mind is: show me all of the log in/out events for some user in some timerange.